toolbox/cijoe-pkg-xnvme/src/cijoe/xnvme/tests/tools/test_zoned.py

```diff
@@ -0,0 +1,222 @@
+"""
+This is a port of the tests:
+
+zoned_append.sh                 --> test_append()
+zoned_changes.sh                --> test_changes()
+zoned_enum.sh                   --> test_enum()
+zoned_idfy_ctrlr.sh             --> test_idfy_ctrlr()
+zoned_idfy_ns.sh                --> test_idfy_ns()
+zoned_info.sh                   --> test_info()
+zoned_mgmt_open.sh              --> MISSING
+zoned_read.sh                   --> test_read()
+zoned_report.sh                 --> test_report_limit()
+zoned_report_all.sh             --> test_report_all()
+zoned_report_one.sh             --> test_report_single()
+zoned_report_some.sh            --> test_report_some()
+zoned_write.sh                  --> test_reset_report_write_report()
+zoned_mgmt_open.sh              --> test_reset_open_report()
```

Comment:
This line should be moved up to line 10 for the alphabetical order to make sense

Reply:
I went with removing all of these comments and adding them to the commit-message instead.